pkg/process/handler.go

```diff
@@ -2,34 +2,175 @@ package process
 
 import (
 	"github.com/AsterNighT/software-engineering-backend/api"
-	"github.com/labstack/echo/v4"
 	"net/http"
 )
 
 type RegistrationHandler struct {
 }
 
 // @Summary get all departments
-// @Tags departments
+// @Tags patient
 // @Description display all departments of a hospital
 // @Produce json
-// @Success 200 {object} api.ReturnedData{data=[]Department}
-// @Router /departments [GET]
+// @Success 200 {array} api.ReturnedData{data=[]Department}
+// @Router /patient/departments [GET]
```

Comment:
That path is a bit strange.
```suggestion
// @Router /departments [GET]
```